public/send-results.js

```diff
@@ -0,0 +1,26 @@
+/* global: location, sessionStorage */
+
+(function() {
+  var path = location.pathname.slice(1)
+
+  if (path === "login") {
+    var submitSurveyButton = document.getElementById('submit-survey-button')
+    var clinicianEmailInput = document.getElementById('clinician-email-input')
+    submitSurveyButton.addEventListener('click', function() {
+      var storeClinicianEmail = sessionStorage.setItem('clinician-email', clinicianEmailInput.value)
+    })
+  }
+
+  if (path === "results") {
+    var sendResultsButton = document.getElementById('send-results-button')
+    var getSessionStorage = window.sessionStorage
+    var getClinicianEmail = getSessionStorage['clinician-email']
```

Comment:
Hi @des-des setItem was used in session-storage.js to store the type of survey

I am using getItem in send-results.js to retrieve the stored value, once the survey has been completed and the results need to be sent to the clinician 

Does that make sense? 

Will change the var name :)